resources/public/api/conf/1.0/headers.raml

```diff
@@ -18,3 +18,17 @@ traits:
         type: string
         required: true
         example: 'application/vnd.hmrc.1.0+json'
+  acceptJson:
```

Comment:
This needs changing to correlationID as acceptJson is erroring

Reply:
done